jssource/src_files/include/SubPanel/SubPanelTiles.js

```diff
@@ -282,16 +282,17 @@ function showSubPanel(child_field, url, force_load, layout_def_key) {
       url = '' + url;
     }
 
-    current_subpanel_url = url;
-    // http_fetch_async(url,got_data,request_id++);
-    var returnstuff = http_fetch_sync(url + '&inline=' + inline + '&ajaxSubpanel=true');
-    request_id++;
-    got_data(returnstuff, inline);
-    $('#whole_subpanel_' + child_field + ' .table-responsive').footable();
-  }
-  else {
-    var subpanel = document.getElementById('subpanel_' + child_field);
-    subpanel.style.display = '';
+		current_subpanel_url = url;
+		// http_fetch_async(url,got_data,request_id++);
+		var returnstuff = http_fetch_sync(url+ '&inline=' + inline + '&ajaxSubpanel=true');
+		request_id++;
+		got_data(returnstuff, inline);
+		if (SUGAR.themes.theme_name=="SuiteP"){$('#whole_subpanel_'+child_field+' .table-responsive').footable();}
+	}
+	else{
+
+		var subpanel = document.getElementById('subpanel_'+child_field);
```

Comment:
It would be better to check it which element exists, and then make a decision based on the element. Check the theme name does not check what is actually on  the page. It provides no guarantee of the state of a subpanel. In general it is best not to check the theme name, as it increases the technical debt.

Reply:
I can not verify (don't have local environment).
But I checked in the console and I see that this solution will not work. *$ ('# whole_subpanel_'+child_field)* in suiteP does not contain classes ('panel panel-default sub-panel'), but this classes contains its child elements.

Your decision on technical debt may be better, but it is worse in understanding the code, which can in the future greatly complicate the debugging. Added a block of implicit behavior that relies on the names of several insignificant class names.

The best solution would be to add some kind of attribute (for example the useFooTable class for *$ ('# whole_subpanel_'+child_field))* to the themes/SuiteP/include/SubPanel/tpls/SubPanelTiles.tpl template and check the fact of using the FooTable plugin in the following way:
``` $('#whole_subpanel_' + child_field).hasClass('useFooTable')```